docs/liquidators/manual.md

```diff
@@ -0,0 +1,162 @@
+# Manual Liquidations
```

Comment:
shall we put this in `/x/leverage/docs`?

Reply:
It's docs for our chain operations, not just a module. We probably need a more central location for that kind of thing

Reply:
The last thing we should have is "15+" (read many) places for documentation. We already have `design_docs` and `x/leverage`. The former is more for discussion, the latter is more about usage. Hence:
* Let's use `x/leverage` for everything `x/leverage` user related
* let's use `docs/...` for everything general (setting up a chain, general requirements).

Reply:
I'll move it to `x/leverage` - having a link in the root readme as well should keep it accessible enough.